var/spack/repos/builtin/packages/sgpp/package.py

```diff
@@ -52,10 +54,22 @@ class Sgpp(SConsPackage):
     # to complain about missing headers (due to a path check not working anymore)
     # See issue https://github.com/SGpp/SGpp/issues/263 and https://github.com/SGpp/SGpp/pull/266
     patch("disable_disutils_deprecation_warning.patch", when="@:3.4.0 ^python@3.10:3.11")
-    # SGpp does not contain aarch64 support as of yet. To make it work still, this patch adds
+    # SGpp does not contain aarch64 support as of 3.4.0. To make it work still, this patch adds
     # simple build system support for it.
-    patch("for_aarch64.patch", when="target=aarch64:")
+    patch("for_aarch64.patch", when="@:3.4.0 target=aarch64:")
```

Comment:
Not requesting a change necessarily. If you plan on making 3.4.1 etc. patch releases that don't have this patch I would recommend this:
```suggestion
    patch("for_aarch64.patch", when="@:3.4 target=aarch64:")
```
However, I see 3.4.0 is the latest release anyway (from quite a while back...), so may not make a difference.

Reply:
Yeah, there is no 3.4.1 release planned at least. The next one should be a 3.5.0 release soon(ish).